sormas-backend/src/main/java/de/symeda/sormas/backend/externaljournal/ExternalJournalService.java

```diff
@@ -137,10 +144,35 @@ private String getPatientDiaryAuthTokenInternal() {
 		}
 	}
 
+	/**
+	 * Notify external journals that a followUpUntilDate has been updated
+	 *
+	 * @param contact
+	 *            a contact assigned to a person already available in the external journal
+	 * 
+	 */
+	public void notifyExternalJournalFollowUpUntilUpdate(ContactDto contact) {
+		SymptomJournalStatus savedStatus = personFacade.getPersonByUuid(contact.getPerson().getUuid()).getSymptomJournalStatus();
```

Comment:
Is it possible that `getPersonByUuid` returns `null` and we get a NPE?

Reply:
I think that is unlikely, because we call this for an existing contact. A contact is always associated with a person. And even if it returns null, the error should already be logged.
Are my reasons convincing enough to leave it like that? If not, what would you like me to do?

Reply:
Yes, it is ok like this